change/@ni-nimble-tokens-7d1352e1-f82c-48b8-94fd-d86131297a6d.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Should clean-out the styledictionary xaml / csharp stuff that was used for the nuget package: https://github.com/ni/nimble/blob/cd78ec74ffdf3d47d405cbbf5cd8ecac6f34a988/packages/nimble-tokens/source/styledictionary/build.js#L55